src/javascript/crypto/e2e/extension/ui/glass/glasswrapper.js

```diff
@@ -93,24 +86,20 @@ ui.GlassWrapper.prototype.installGlass = function(opt_callback) {
   glassFrame.scrolling = 'no';
   goog.style.setSize(glassFrame, goog.style.getSize(this.targetElem_));
   glassFrame.style.border = 0;
-  var pgpMessage = '';
-  var surroundings = ['', ''];
-
-  if (this.originalText_) {
-    pgpMessage = this.originalText_;
-  } else {
-    pgpMessage =
-        e2e.openpgp.asciiArmor.extractPgpBlock(this.targetElem_.innerText);
-    surroundings = this.targetElem_.innerText.split(pgpMessage);
-  }
+
+  var pgpMessage =
+      e2e.openpgp.asciiArmor.extractPgpBlock(this.targetElem_.innerText);
+  var surroundings = this.targetElem_.innerText.split(pgpMessage);
```

Comment:
i'm forwarding my email to you too for your examination (i operated my local key server).

Should I merge this PR as is? You may like to make a more resilient patch later.